src/pages/PersonalInfo/PersonalInfo.module.css

```diff
@@ -0,0 +1,44 @@
+.tabs {
+  -webkit-tap-highlight-color: transparent;
+  width: 100%;
+  font-family: 'Inter', serif;
+}
+
+.tabList { /*applies to the entire box of tabs */
+  border-bottom: 1px solid gray;
+  margin: 0 0 10px; /*top right/left bottom-applies to separate the text for each tab from the tablist */
+  padding: 0;
+  width: 100%;
+  flex-direction: row;
+  justify-content: center;
+  display: flex;
+}
+
+.tab {
+  /*applies to each individual tab */
+  border: none;
+  color: gray;
+  background: transparent;
+  margin: 10px 20px;
+  font-size: 90%;
+  padding: 6px 12px;
+  cursor: pointer;
+}
+
+.tabSelected {
+  border: none;
+  border-bottom: 2px solid #cf2e2e;
```

Comment:
Just a note about this, we will be making CSS variable that will replace the hardcoded colors soon

Reply:
ok